lychee-lib/src/filter/excludes.rs

```diff
@@ -1,8 +1,7 @@
 use regex::RegexSet;
-use std::net::IpAddr;
-
-use crate::Uri;
 
+// TODO: renaming
+// TODO: should just be added to the Excludes set.
```

Comment:
That's a great idea! We could call it `default_excludes` or something

Reply:
In a second thought, this is doable but will makes the filtering messy.

Because when only `includes` are provided without `excludes`, we require URLs to be explicitly included.

If we add false positives as default excludes, this mechanism need to be changed.

So the updated code removed these TODOs.